scripts/local-coder/lib/bmalph.sh

```
@@ -0,0 +1,114 @@
+    cs_bmalph_load_defaults
+    export PATH="${CS_USER_NPM_GLOBAL_BIN}:${PATH}"
+    cs_require_command bmalph || return 1
+
+    expected_marker="$(cs_bmalph_expected_platform_marker "${platform}")"
+    tmp_project_dir="$(mktemp -d)"
+    tmp_output="$(mktemp)"
+
+    # Remove temporary files created during dry-run verification.
+    cleanup() {
+        rm -rf "${tmp_project_dir}" "${tmp_output}"
+    }
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)